apps/src/templates/courseOverview/CourseEditor.js

```diff
@@ -98,14 +112,30 @@ export default class CourseEditor extends Component {
             ))}
           </select>
         </label>
+        <VisibleInTeacherDashboard
+          paramName="visible"
+          checked={this.state.visible}
+          disabled={!!this.state.pilotExperiment}
+          onChange={e => this.setState({visible: e.target.checked})}
+        />
         <label>
-          Pilot Experiment
+          Can be recommended (aka stable)
           <input
-            name="pilot_experiment"
-            defaultValue={pilotExperiment}
-            style={styles.input}
+            name="is_stable"
+            type="checkbox"
+            defaultChecked={this.props.isStable}
+            style={styles.checkbox}
           />
+          <p>
+            If checked, this unit will be eligible to be the recommended version
```

Comment:
I think this should say course instead of unit?  "If checked, this course will be eligible to be the recommended version of the course. The most recent eligible version will be the recommended version."

Reply:
We should probably add this string to i18n list

Reply:
On course / unit - yea, copy paste error. I'll fix.

On adding string to i18n list - what does that mean exactly?

Reply:
Nevermind Dave said we didn't need to worry about it here.